lib/taskana-core/src/test/java/acceptance/task/complete/CompleteTaskAccTest.java

```diff
@@ -1,118 +1,159 @@
+        .permission(WorkbasketPermission.APPEND)
+        .buildAndStore(workbasketService);
+
+    defaultObjectReference = defaultTestObjectReference().build();
+  }
 
   @WithAccessId(user = "user-1-1")
   @Test
-  void testCompleteTask() throws Exception {
-    assertThat(taskService.getTask("TKI:000000000000000000000000000000000001").getState())
-        .isEqualTo(TaskState.CLAIMED);
-    Task completedTask = taskService.completeTask("TKI:000000000000000000000000000000000001");
+  void should_CompleteClaimedTask() throws Exception {
+    Task claimedTask = createTaskClaimedByUser().buildAndStore(taskService);
+
+    Task completedTask = taskService.completeTask(claimedTask.getId());
+
     assertThat(completedTask).isNotNull();
     assertThat(completedTask.getCompleted()).isNotNull();
     assertThat(completedTask.getState()).isEqualTo(TaskState.COMPLETED);
-    assertThat(completedTask.getModified()).isNotEqualTo(completedTask.getCreated());
+    assertThat(completedTask.getCompleted())
+        .isEqualTo(completedTask.getModified())
+        .isNotNull()
+        .isNotEqualTo(completedTask.getCreated());
   }
 
   @WithAccessId(user = "admin")
   @Test
-  void should_completeClaimedTaskByAnotherUser_When_UserIsAdmin() throws Exception {
-    assertThat(taskService.getTask("TKI:000000000000000000000000000000000029").getState())
-        .isEqualTo(TaskState.CLAIMED);
-    Task completedTask = taskService.completeTask("TKI:000000000000000000000000000000000029");
+  void should_CompleteClaimedTaskByAnotherUser_When_UserIsAdmin() throws Exception {
+    Task claimedTask = createTaskClaimedByUser().buildAndStore(taskService);
+
+    Task completedTask = taskService.completeTask(claimedTask.getId());
+
     assertThat(completedTask).isNotNull();
     assertThat(completedTask.getCompleted()).isNotNull();
     assertThat(completedTask.getState()).isEqualTo(TaskState.COMPLETED);
-    assertThat(completedTask.getModified()).isNotEqualTo(completedTask.getCreated());
+    assertThat(completedTask.getCompleted())
+        .isEqualTo(completedTask.getModified())
+        .isNotNull()
+        .isNotEqualTo(completedTask.getCreated());
   }
 
   @WithAccessId(user = "admin")
   @WithAccessId(user = "taskadmin")
   @TestTemplate
-  void should_ForceCompleteTask_When_NoExplicitPermissionsButUserIsInAdministrativeRole()
+  void should_ForceCompleteClaimedTask_When_NoExplicitPermissionsButUserIsInAdministrativeRole()
       throws Exception {
-    resetDb(false);
+    Task claimedTask = createTaskClaimedByUser().buildAndStore(taskService);
+
+    Task completedTask = taskService.forceCompleteTask(claimedTask.getId());
 
-    assertThat(taskService.getTask("TKI:000000000000000000000000000000000000").getState())
-        .isEqualTo(TaskState.CLAIMED);
-    Task completedTask = taskService.forceCompleteTask("TKI:000000000000000000000000000000000000");
     assertThat(completedTask).isNotNull();
     assertThat(completedTask.getCompleted()).isNotNull();
```

Comment:
since the assert statements two lines below already contains `isNotNull` this is not necessary.